content/License.txt

```diff
@@ -1,58 +0,0 @@
-
```

Comment:
This file must not be deleted. It is being linked from: https://www.nuget.org/packages/Libuv/1.9.0 and https://www.nuget.org/packages/Libuv/1.9.1